bolt-servlet/src/test/java/samples/OpenIDConnectSample.java

```diff
@@ -49,6 +49,10 @@ public static void main(String[] args) throws Exception {
                     OpenIDConnectUserInfoResponse userInfo = teamIdWiredClient.openIDConnectUserInfo(r -> r
                             .token(refreshedToken.getAccessToken()));
                     logger.info("userInfo: {}", userInfo);
+
+                    // To revoke this, you can use auth.revoke API method
+                    // Slack.getInstance().methods().authRevoke(r -> r.token(refreshedToken.getAccessToken()));
```

Comment:
I did a manual test using this example app. It's not so easy to write meaningful unit tests for this pattern.

Reply:
I take your word for it! It would be nice to test that `lookupOrResolve` isn't invoked if belonging to the skipped methods, but if that was possible I'd imagine you'd have included it.

Reply:
Good point! Indeed, it would be possible to write such simple test if we refactor the internals of the `MethodsClientImpl` (meaning, enable to pass `TeamIdCache` in some way -- currently it's private with final keyword). But I would like to avoid changing the internals a lot only for this purpose. When we see the need to write the test next time, it'd be a good timing to consider making the part more unit-testable.